.github/workflows/publish.yaml

```diff
@@ -0,0 +1,61 @@
+# ./.github/workflows/publish.yml
+name: Publish
```

Comment:
We now have automatic publishing, need to work out the kinks and then will write a guide on how to release stuff